### DISCUSSION TITLE
Toe stud placement

### DISCUSSION THREAD

**Discussion by newVideophile** [GrandOrgue/grandorgue#1712] [General]
This is not a GO specific topic, but since there are many organists and organ builders here, I'd like some advice regarding the placement of a toe stud.

I currently have only pistons under the great manual to recall memories and change registration on the fly as I play. But I am now learning the Widor toccata, and both hands are continuously busy, so the only option is a toe stud, programmed as a "Next" piston. 

My question is: where should I place it? My organ has a single expression pedal, off-centered to the right, so I would tend to place the toe stud on the left of the expression pedal, corresponding to the center or the pedalboard. This way, I could use it with either my left or right foot, depending on which is available. 

But [this post](https://organforum.com/forums/forum/pipe-organs/classic-church-pipe-organs/738340-next-toe-stud-placement) confuses me. There seems to be a consensus that it should be placed on the right of the expression pedal(s). I don't understand this, because if my right foot is busy, then I won't have access to the toe stud. 

Any advice?

**Comment by midimusic** [comment]
@newVideophile 

Since nobody else seems to have any ideas on this - I suggest you install two toe studs. One to be placed for  the left foot and the other to be placed for the right foot. The two can probably be wired in parallel  so there is minimal additional complexity.

csw900

**Reply by newVideophile** [reply]
Thank you for your reply and your wise recommendation. After deeper analysis of the Toccata, I also came to the conclusion that I need (at least) two toe studs. There is a passage where I need to press the toe stud and quickly after use both feeth in the very upper part of the pedalboard...

**Comment by oleg68** [comment]
I use two toe studs: the right one is mapped to `Next` and the left one is mapperd to `Prev`.

I also have the sequencer panel open on the right screen, so when my feet are busy I am sitill able to switch between combination by finger on the touchscreen.

Of cause, when both hands and feet all are busy, I am not able to switch combinations at all.

**Comment by newVideophile** [comment]
Yes, your setup seems quite common, but I don't understand its rationale.

First, I don't see any benefit for a Prev button. Except perhaps if you hit Next by mistake. But I can't see this happen, because there is nothing else to hit or press around it. And I will program it with a very high anti-bounce time to avoid accidental double press. Then, if you right foot is busy, how do you manage to press the Next button with your left foot if the toe stud is on the right?

So I am currently leaning towards 2 Next buttons, if not 3.

But I may be missing something. I started learning organ only 2 1/2 years ago, I still consider myself as a beginner

**Reply by oleg68** [reply]
>  I don't see any benefit for a Prev button. Except perhaps if you hit Next by mistake. 

Yes, sometimes I press the Next button by mistake.

Sometimes, when I learn some piece, I need to return to the previous part or to the beginning. But mostly I do this with the touchscreen rather the toe stud.

> if you right foot is busy, how do manage to press the Next button with your left foot if the toe stud is on the right?

No, I do not. I can press Next only by the right foot or by a finger of the right hand on the touchscreen.

**Reply by newVideophile** [reply]
Ok, I understand. But I challenge you to play the Widor toccata with your setup then ;-)

Could you share a photo of your two toe studs?

**Reply by oleg68** [reply]
This image is not a very good, but look here: 
https://github.com/GrandOrgue/grandorgue/discussions/875#discussioncomment-7564171
